contracts/src/v0.8/ccip/offRamp/OffRamp.sol

```diff
@@ -95,7 +97,7 @@ contract OffRamp is ITypeAndVersion, MultiOCR3Base {
   // solhint-disable-next-line gas-struct-packing
   struct StaticConfig {
     uint64 chainSelector; // ───╮  Destination chainSelector
-    IRMNV2 rmn; // ─────────────╯  RMN Verification Contract
+    IRMNRemote rmnRemote; // ───╯  RMN Verification Contract
```

Comment:
@0xnogo is this a breaking change for your RMN remote reader?

Reply:
It is not since we are getting the RMN remote address from the offramp static config getter. But I noticed that its named changed, so we will have to update that (but unrelated to this PR change).